uaclient/config.py

```diff
@@ -56,6 +56,7 @@ class UAConfig:
         ),
         "machine-id": DataPath("machine-id", True),
         "machine-token": DataPath("machine-token.json", True),
+        "resources": DataPath("resources.json", False),
```

Comment:
Why do we need to cache `resources`? Shouldn't the `status-cache` be sufficient for the non-root user case?

Reply:
Dan resources isn't strictly needed, it is just that I've UAContract client caches the unaltered response of any Contract api call in private/<some-cache> If we are relying solely on the adapted response in status-cache, then future us will have to recall that this is not strictly what the contract server returned (or what keys were provided in that response).  I *think* we decided in the past that our debug log which captures this actual response is reasonbale enough, it just makes future triage a bit harder if contract backend changes response and it breaks us.

Reply:
Yeah, I think the log is the way to go here. Having the file on disk without any context about how we got that content is less useful than what we have in the log, so even for responses we cache, I think that would be the first place I'd go when debugging.

Reply:
good point, dropped.